support/machine.go

```diff
@@ -30,3 +30,15 @@ func GetMachines(t Test, machineSetName string) []machinev1beta1.Machine {
 func MachineSetId(machineSet machinev1beta1.MachineSet) string {
 	return machineSet.Name
 }
+
+func MachineSet(test Test, namespace string, machineSetName string) func(g gomega.Gomega) *machinev1beta1.MachineSet {
```

Comment:
Nit pick - you could use t here to match other functions for uniformity in the file.
```suggestion
func MachineSet(t Test, namespace string, machineSetName string) func(g gomega.Gomega) *machinev1beta1.MachineSet {
```